### ISSUE TITLE
Uncaught Error: Cannot find module '$ATOM_HOME/packages/Hydrogen/node_modules/ansi-to-react/lib/i...

### ISSUE THREAD

**Issue by liyue34673** [nteract/hydrogen#2031]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.53.0 x64
**Electron**: 6.1.12
**OS**: Mac OS X 10.15.7
**Thrown From**: [Hydrogen](https://github.com/nteract/hydrogen) package 2.14.6

### Stack Trace

Uncaught Error: Cannot find module '/Users/yli/.atom/packages/Hydrogen/node_modules/ansi-to-react/lib/index.js'. Please verify that the package.json has a valid "main" entry

```
At internal/modules/cjs/loader.js:240

Error: Cannot find module '/Users/yli/.atom/packages/Hydrogen/node_modules/ansi-to-react/lib/index.js'. Please verify that the package.json has a valid "main" entry
    at tryPackage (internal/modules/cjs/loader.js:232:19)
    at Function.Module._findPath (internal/modules/cjs/loader.js:369:18)
    at Module._resolveFilename (internal/modules/cjs/loader.js:614:27)
    at Module._resolveFilename (/Applications/Atom.app/Contents/Resources/electron.asar/common/reset-search-paths.js:41:16)
    at Function.get_Module._resolveFilename (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:155481)
    at Module.require (/app.asar/static/index.js:61:43)
    at require (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:146814)
    at /packages/Hydrogen/node_modules/@nteract/outputs/lib/components/kernel-output-error.js:13:41)
    at /packages/Hydrogen/node_modules/@nteract/outputs/lib/components/kernel-output-error.js:44:3)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:147498)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:151046)
    at Module.load (internal/modules/cjs/loader.js:645:32)
    at Function.Module._load (internal/modules/cjs/loader.js:560:12)
    at Module.require (/app.asar/static/index.js:72:46)
    at require (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:146814)
    at /packages/Hydrogen/node_modules/@nteract/outputs/lib/index.js:14:29)
    at /packages/Hydrogen/node_modules/@nteract/outputs/lib/index.js:27:3)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:147498)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:151046)
    at Module.load (internal/modules/cjs/loader.js:645:32)
    at Function.Module._load (internal/modules/cjs/loader.js:560:12)
    at Module.require (/app.asar/static/index.js:72:46)
    at require (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:146814)
    at /packages/Hydrogen/lib/components/inspector.js:4:38)
    at /packages/Hydrogen/lib/components/inspector.js:48:25)
    at Module.get_Module._compile (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:147498)
    at Object.value [as .js] (/Applications/Atom.app/Contents/Resources/app/static/<embedded>:11:151046)
    at Module.load (internal/modules/cjs/loader.js:645:32)
    at Function.Module._load (internal/modules/cjs/loader.js:560:12)
    at Module.require (/app.asar/static/index.js:72:46)
```

### Commands

```
     -8:23.8.0 core:move-down (input.hidden-input)
     -8:23.1.0 editor:newline (input.hidden-input)
     -8:18.4.0 hydrogen:run-and-move-down (input.hidden-input)
  8x -5:52.9.0 core:backspace (input.hidden-input)
     -5:50 editor:newline (input.hidden-input)
     -3:53.2.0 core:copy (atom-notification.fatal.icon.icon-bug.native-key-bindings.has-detail.has-close.has-stack)
```

### Non-Core Packages

```
atom-html-preview 0.2.6 
autocomplete-python 1.16.0 
build-npm-apm 0.12.0 
file-icons 2.1.45 
Hydrogen 2.14.6 
lodash-snippets 2.0.0 
python-autopep8 0.1.3 
script 3.26.0 
```

**Comment by royceschultz** [comment]
See #1945 
TLDR: For some reason the installation is missing the lib folder. Go to [this guy's](https://github.com/nteract/hydrogen/issues/1945#issuecomment-636422322) github, download the lib folder and place it in /Users/yli/.atom/packages/Hydrogen/node_modules/ansi-to-react/ (the location referenced in the error).

**Comment by royceschultz** [comment]
Not sure if this is related. After installing ipykernel and following [this instruction](https://github.com/udacity/aind2-dl/issues/9#issuecomment-562692658) because I'm on a mac, I tried printing using hydrogen and got the following error:
`Uncaught Invariant Violation: Minified React error #130; visit http://facebook.github.io/react/docs/error-decoder.html?invariant=130&args[]=undefined&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
events.js:160`

Installing an older version of hydrogen per [this comment](https://github.com/nteract/hydrogen/issues/1945#issuecomment-639646483) fixed it for me. Run this in your terminal:
`apm install hydrogen@2.9.0`

**Comment by liyue34673** [comment]
> Not sure if this is related. After copying the lib folder mentioned in my comment above, installing ipykernel and following [this instruction](https://github.com/udacity/aind2-dl/issues/9#issuecomment-562692658) because I'm on a mac, I tried printing using hydrogen and got the following error:
> `Uncaught Invariant Violation: Minified React error #130; visit http://facebook.github.io/react/docs/error-decoder.html?invariant=130&args[]=undefined&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings. events.js:160`
> 
> Installing an older version of hydrogen per [this comment](https://github.com/nteract/hydrogen/issues/1945#issuecomment-639646483) fixed it for me. Run this in your terminal:
> `apm install hydrogen@2.9.0`

Thanks sooo much! I got this error too and fixed that under your guidance!!
